src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -15,7 +15,9 @@
  * Annotation class for @Route().
  *
  * @Annotation
+ *
  * @NamedArgumentConstructor
+ *
```

Comment:
It seems that we don't want empty lines here: https://github.com/symfony/symfony/pull/52389

Reply:
Oh, fabbot wants to add this newline. That's annoying.

Reply:
If I remember correctly, we ignored those precise fabbot fixes in the past 🙂